tee-worker/omni-executor/aa-contracts/src/interfaces/OwnerType.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+/* solhint-disable no-inline-assembly */
+
+/**
+ * The owner type
+ * Required as a single byte in the `OmniAccount.initialize` to allow for more fine-grained access control
+ *
+ * See P-1644 for more context
+ *
+ * The fields of this enum are 1 to 1 mapped to the rust `UserId` type in tee-worker/omni-executor/executor-primitives/src/auth.rs
+ */
+enum OwnerType {
```

Comment:
I just wanted to confirm that we keep prefix [here](https://github.com/litentry/heima/blob/dev/tee-worker/omni-executor/aa-contracts/src/accounts/OmniAccount.sol#L97) unchanged, right ?